### PR TITLE
Added Personalization brochure

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | master, 4+
| Edition       | 

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/personalization/personalization_brochure.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Personalization
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:1]:
```suggestion
# Personalization product guide
```

File: docs/personalization/personalization_brochure.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## What is Personalization
+
+Personalization engine is a cloud-based service that tracks and analyzes customer behaviors and delivers optimized, user-tailored content, and information.
+
+The engine uses artificial intelligence and machine learning modelling to learn and create behavioral patterns and generate recommendations ready to be presented on one or more websites hosted by the Ibexa DXP instance.
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:7]:
```suggestion
The engine uses artificial intelligence and machine learning modeling to learn and create behavioral patterns and generate recommendations ready to be presented on one or more websites hosted by the [[= product_name =]] instance.
```

File: docs/personalization/personalization_brochure.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+## Availability
+
+Personalization service is available in all supported Ibexa DXP versions and editions.
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:11]:
```suggestion
Personalization service is available in all supported [[= product_name =]] versions and editions.
```

File: docs/personalization/personalization_brochure.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+Pricing is based on the number of Personalization API events.
+
+To enable recommendation engine, request access to the service by [creating an account](https://doc.ibexa.co/projects/userguide/en/master/personalization/enable_personalization/#request-access-to-the-server).
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:15]:
```suggestion
To enable recommendation engine, request access to the service by [creating an account]([[= user_doc =]]/personalization/enable_personalization/#request-access-to-the-server).
```

File: docs/personalization/personalization_brochure.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+Pricing is based on the number of Personalization API events.
+
+To enable recommendation engine, request access to the service by [creating an account]([[= user_doc =]]/personalization/enable_personalization/#request-access-to-the-server).
+
+After setting up an account and receiving credentials, pass them to your administrator who can configure Personalization service in your [[= product_name =]] instance.
+
+Now, you can start collecting data and boost your business.
+
+## How does Personalization work?
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:21]:
```suggestion
## How does Personalization work
```
you don't have question marked in the first header

File: docs/personalization/personalization_brochure.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+### Models
+
+Models are statistics-based and perform calculations based on data from content, users and events.
+
+They work in the background and update on regular time intervals to ensure best and accurate recommendations. Personalization service includes a few predefined models according to your needs and agreements between your organization and Ibexa.
+
+If you need more customized models, contact customer support.
+
+Includes the following model types:
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:45]:
```suggestion
Personalization service includes the following model types:
```

File: docs/personalization/personalization_brochure.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+For more information see documentation for API operations available for Personalization:
+
+[Tracking API](api_reference/tracking_api.md/)
+
+[Content API](api_reference/content_api.md/)
+
+[Recommendation API](api_reference/recommendation_api.md/)
+
+[User API](api_reference/user_api.md/)
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:83]:
```suggestion
- [Tracking API](api_reference/tracking_api.md/)
- [Content API](api_reference/content_api.md/)
- [Recommendation API](api_reference/recommendation_api.md/)
- [User API](api_reference/user_api.md/)
```

File: docs/personalization/personalization_brochure.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+### Reduce clutter and improve customer retention
+
+Help users find content of their interest quicker
+
+use words to appeal to individual customers, as well as segments
+
+Enrich customer data by integrating Ibexa Personalization with other systems such as ERP, CRMs.
+
+Track user’s scenarios and develop strategies that match your strategies and improve engagement
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:99]:
```suggestion
Help users find content of their interest quicker,
use words to appeal to individual customers, as well as segments.
Enrich customer data by integrating Ibexa Personalization with other systems such as ERP, CRMs.
Track user’s scenarios and develop strategies that match your strategies and improve engagement.
```

File: docs/personalization/personalization_brochure.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+
+### Satisfy customer expectations
+
+Personalization engine shows customers products relevant to their interests, builds a feeling of connection and understanding
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:104]:
```suggestion
Personalization engine shows customers products relevant to their interests, builds a feeling of connection and understanding.
```

File: docs/personalization/personalization_brochure.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+Connect Personalization engine with your eCommerce shop, focus on the revenue generated by recommendations. Track what recommendations are shown to your visitors and analyze conversion rates. Effectively evaluate quality of recommendations and compare with your KPIs.
+
+## Use cases
+
+[eCommerce]([[= user_doc =]]/personalization/use_cases/#ecommerce.md)
+
+[Content publishing]([[= user_doc =]]/personalization/use_cases/#content-publishing.md)
+
+[Multiple website hosting]([[= user_doc =]]/personalization/use_cases/#multiple-website-hosting.md)
```
Comment (MagdalenaZuba) [inline docs/personalization/personalization_brochure.md:131]:
```suggestion
- [eCommerce]([[= user_doc =]]/personalization/use_cases/#ecommerce.md)
- [Content publishing]([[= user_doc =]]/personalization/use_cases/#content-publishing.md)
- [Multiple website hosting]([[= user_doc =]]/personalization/use_cases/#multiple-website-hosting.md)
```

File: mkdocs.yml [hunk 11]
```diff
@@ -399,6 +399,7 @@ nav:
         - Importing historical user tracking data: personalization/importing_historical_user_tracking_data.md
         - Track with ibexa-tracker.js: personalization/tracking_with_ibexa-tracker.md
         - Attribute search in Elasticsearch: personalization/attribute_search_in_elasticsearch.md
+        - Personalization guide : personalization/personalization_brochure.md
```
Comment (MagdalenaZuba) [inline mkdocs.yml:402]: I would place it under "How Personalization works" or even replace it as this guide describes it better and we don't need the repetition.

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 11 above)
Comment (juskora) [reply mkdocs.yml:402]: I would leave How personalization works anyway.

**Review by MagdalenaZuba — APPROVED** [review]

File: docs/personalization/personalization_brochure.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+Create websites with content targeted at your customers. Deliver your visitors relevant content and build trust in your brand.
+
+### Reduce clutter and improve customer retention
+
+Help users find content of their interest quicker,
+use words to appeal to individual customers, as well as segments.
+Enrich customer data by integrating Ibexa Personalization with other systems such as ERP, CRMs.
+Track user’s scenarios and develop strategies that match your strategies and improve engagement.
+
+
```
Comment (julitafalcondusza) [inline docs/personalization/personalization_brochure.md:95]: unnecessary empty line

**Review by julitafalcondusza — APPROVED** [review]

File: docs/personalization/personalization_brochure.md (hunk 12 above)
Comment (juskora) [inline docs/personalization/personalization_brochure.md:95]:
```suggestion

```

**Pull request merged**